benchmark/runner/tpch_runner.cpp

```diff
@@ -13,21 +13,11 @@ class TPCHRunner : public benchmark::Fixture {
   const uint64_t execution_us_per_worker_ = 1000000;  // Time (us) to run per terminal (worker thread)
   std::vector<uint64_t> avg_interval_us_ = {10, 20, 50, 100, 200, 500, 1000};
   const execution::vm::ExecutionMode mode_ = execution::vm::ExecutionMode::Interpret;
+  uint32_t query_num_ = 9;
 
   std::unique_ptr<DBMain> db_main_;
   std::unique_ptr<tpch::Workload> tpch_workload_;
 
-  // TPCH setup
-  const std::vector<std::string> tpch_query_filenames_ = {
-      "../../../tpl_tables/sample_tpl/tpch_q1.tpl",
-      "../../../tpl_tables/sample_tpl/tpch_q4.tpl",
-      "../../../tpl_tables/sample_tpl/tpch_q5.tpl",
-      "../../../tpl_tables/sample_tpl/tpch_q6.tpl",
-      "../../../tpl_tables/sample_tpl/tpch_q7.tpl",
-      "../../../tpl_tables/sample_tpl/tpch_q11.tpl",
-      "../../../tpl_tables/sample_tpl/tpch_scan_lineitem.tpl",
-      "../../../tpl_tables/sample_tpl/tpch_scan_orders.tpl",
-  };
   const std::string tpch_table_root_ = "../../../tpl_tables/tables/";
   const std::string tpch_database_name_ = "tpch_db";
 
```

Comment:
I cannot comment on that line... but please change
`metrics_manager->EnableMetric(metrics::MetricsComponent::EXECUTION, 0);`
to
`metrics_manager->EnableMetric(metrics::MetricsComponent::EXECUTION_PIPELINE, 0);`

We will directly generate the features with the new benchmark since we have the query plans. Please change this and see whether the benchmark runs. You should see a file called `pipeline.csv` generated by the program with the features and metrics.

Reply:
As per discussion, EXECUTION_PIPELINE still has bugs now

Reply:
Fixed